### ISSUE TITLE
'data/data.pickle' file not found.

### ISSUE THREAD

**Issue by rushin682** [dev-td7/PyStuck#1]
Got the following error when executing prepare_data.py file. Installed all the dependencies as intructed in the requirements. Could you help please.

Traceback (most recent call last):                
  File "prepare_data.py", line 88, in <module>
    dump_data()
  File "prepare_data.py", line 65, in dump_data
    pickle.dump(dump, open('data/data.pickle', 'wb'), protocol=-1)
FileNotFoundError: [Errno 2] No such file or directory: 'data/data.pickle'

**Comment by dev-td7** [comment]
Thanks! I have added the '+' sign in write mode to automatically create the pickle file if it doesn't exist. Also, I am pushing a new version for preparing data files. It optimizes RAM usage, as the current one directly loads all data into RAM, and is inefficient with respect to space. I hope this solves the error.

Also, what OS are you using? I have found that running this on Windows may cause MemoryError.
